Src/BootCamp.Chapter/Algorithm.cs

```diff
@@ -27,19 +27,22 @@ internal static void OptionalTestCases()
         /// <returns>The sorted array.</returns>
         internal static int[] Sort(int[] array, ref int swapCount)
         {
-            if (array.Length > 2)
+            if (array != null && array.Length > 0)
             {
-                Split(array, out int[] array1, out int[] array2);
-                return Merge(Sort(array1, ref swapCount), Sort(array2, ref swapCount));
+                if (array.Length > 2)
+                {
+                    Split(array, out int[] array1, out int[] array2);
+                    return Merge(Sort(array1, ref swapCount), Sort(array2, ref swapCount));
+                }
+                if (array.Length == 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5052333)